### ISSUE TITLE
Developer website is out of date

### ISSUE THREAD

**Issue by JAnderton** [kuona/kuona.github.io#2]
Current deployed version of the [dev guide](https://kuona.github.io/developer/) is out of date

**Comment by grahambrooks** [comment]
Good point. I should redirect people here http://kuona.io

**Comment by xuv** [comment]
Indeed. Trying to install Kuona, it got me going in circles with this repo: https://github.com/kuona/kuona which I have now found out it's not the right one.
